Quizzler-iOS13/Model/QuizManager.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  QuizManager.swift
+//  Quizzler-iOS13
+//
+//  Created by Ronald Ngounou on 4/18/24.
+//  Copyright © 2024 The App Brewery. All rights reserved.
+//
+
+import Foundation
+
+class QuizManager{
+    
+    var questions: [ChoicesQuestions] = []
+    var questionNumber = 0
+    var score = 0
+    
+    init() {
+        let question1 = ChoicesQuestions(questionText: "What is the capital of France?", choices: ["London", "Paris", "Berlin"], correctAnswerIndex: 1, explanation: "Paris is the capital of France.")
```

Comment:
[note] We should not have any logic on init, that's anti-pattern. You can keep it like this for now and refactor later

Reply:
Alright!